examples/autoqasm/6_Customize_gate_calibrations.ipynb

```diff
@@ -0,0 +1,404 @@
+      "Help on function rx in module braket.experimental.autoqasm.instructions.gates:\n",
+      "\n",
+      "rx(target: Union[int, oqpy.classical_types._ClassicalVar, oqpy.base.OQPyExpression, str, oqpy.quantum_types.Qubit], angle: float) -> None\n",
+      "    X-axis rotation gate.\n",
+      "    \n",
+      "    Args:\n",
+      "        target (QubitIdentifierType): Target qubit.\n",
+      "        angle (float): Rotation angle in radians.\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "help(rx)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a1189626",
+   "metadata": {},
+   "source": [
+    "The pulse operations of the gate calibration is composed in the body of a function decorated by `@aq.gate_calibration`. To specify fixed values for the parameters `target` and `angle`, we set the values through keyword arguments of the decorator."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 43,
+   "id": "64ebd3a2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wf = DragGaussianWaveform(\n",
+    "    24e-9, 2.547965400864e-9, 2.370235498840002e-10, 0.293350447987059, False, \"wf_drag_gaussian_1\"\n",
+    ")\n",
+    "q0_rf_frame = device.frames['q0_rf_frame']\n",
+    "\n",
+    "@aq.gate_calibration(implements=rx, target=\"$0\", angle=np.pi/2)\n",
+    "def my_defcal():\n",
+    "    pulse.barrier(\"$0\")\n",
+    "    pulse.shift_frequency(q0_rf_frame, -321047.14178613486)\n",
+    "    pulse.play(q0_rf_frame, wf)\n",
+    "    pulse.shift_frequency(q0_rf_frame, 321047.14178613486)\n",
+    "    pulse.barrier(\"$0\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9300f0f8",
+   "metadata": {},
+   "source": [
+    "After the gate calibration is defined, it is ready to be attach to a quantum program. Say, we want to attach `my_defcal` to the main quantum program `my_program`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 53,
+   "id": "bae7731a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "OPENQASM 3.0;\n",
+      "rx(pi / 2) $0;\n",
+      "rz(0.123) $1;\n",
+      "bit __bit_0__;\n",
+      "__bit_0__ = measure $0;\n"
+     ]
+    }
+   ],
+   "source": [
+    "@aq.main\n",
+    "def my_program():\n",
+    "    rx(\"$0\", np.pi/2)\n",
+    "    rz(\"$1\", 0.123)\n",
+    "    measure(\"$0\")\n",
+    "\n",
+    "main_program = my_program()\n",
+    "print(main_program.to_ir())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c7579516",
+   "metadata": {},
+   "source": [
+    "We can use the `with_calibrations` method of the program which creates a new program with `my_defcal` attached, leaving the original intact. You can experiment with different gate calibrations on the same program without needing to redefine the main program every time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 54,
+   "id": "6b7cc158",
+   "metadata": {},
+   "outputs": [
```

Comment:
If you do print the OpenQASM, explain why you're printing it. What should the reader notice?